portfolio-project-matching-app/components/ApplicationCard.js

```diff
@@ -29,7 +31,7 @@ const ApplicationCard = ({ appId, isOutgoing }) => {
         const updatedApp = await readObjectById('applications', app.id, true)
         // handle case of approved application
         if (response === 'Approved') {
-            createAssociation('projects_users', app.project_id, app.user_id)
+            createAssociation('projects_users', app.projectId, app.userId)
```

Comment:
simply used the wrong fields here. I really should normalize the whole `projectId` `project_id` thing between the DB and the models used in the `.js` files.

Once again, poor planning on my part... should be the same name format across the board for simplicity.